plugins/kubectl-appstatus

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
You have made this file executable, right? (chmod +x kubectl-appstatus)

Reply:
Yes.